src/sentry/utils/audit.py

```diff
@@ -1,8 +1,12 @@
 from __future__ import absolute_import
-from sentry.models import ApiKey, AuditLogEntry
+from sentry.models import (
+    ApiKey, AuditLogEntry, AuditLogEntryEvent, DeletedOrganization, DeletedProject, DeletedTeam, Organization, Project, Team
+)
 
 
 def create_audit_entry(request, transaction_id=None, logger=None, **kwargs):
+    """ Creates an audit entry if the given request includes an event
```

Comment:
tbh I don't know what this comment means and I don't think it's making anything more readable. I'd either remove this comment or make it more correct. The comment seems to not really explain the correct behavior. Maybe itemize what actions are performed?

Reply:
Ah I see. I can remove it. I think I meant to come back to the comments on this file, but  didn't end up doing so. My apologies.